test.py

```diff
@@ -24,6 +24,10 @@
 # unresponsive for 5 minutes the parent will presume it dead / incapacitated.)
 TIMEOUT = 300  # Seconds
 
+# Skip this list of unit tests. One reason may be that the original batch size
+# used in the paper is too large to fit on the CI's GPU.
+TRAIN_BLACKLIST = {("densenet121", "cuda")}
```

Comment:
This has to be `TRAIN_EXCLUDELIST`, because the powers that be have decided blacklist is no longer allowed to be used.

Reply:
Good catch, thanks fixed it!